includes/Classifai/Providers/OpenAI/DallE.php

```diff
@@ -149,15 +142,65 @@ public function render_provider_fields() {
 			[
 				'option_index'  => static::ID,
 				'label_for'     => 'style',
-				'options'       => [
-					'vivid'   => __( 'Vivid', 'classifai' ),
-					'natural' => __( 'Natural', 'classifai' ),
-				],
+				'options'       => self::get_image_style_options(),
 				'default_value' => $settings['style'],
 				'description'   => __( 'The style of the generated images. Vivid causes more hyper-real and dramatic images. Natural causes more natural, less hyper-real looking images.', 'classifai' ),
 				'class'         => 'classifai-provider-field hidden provider-scope-' . static::ID,
 			]
 		);
+
+		add_settings_field(
```

Comment:
With settings now managed in JS, we don't need to use `add_settings_field` anymore. We left that in place for existing settings but for new settings, I suggest we don't add this as it isn't actually used

Reply:
The one thing we do need though that I'm not seeing in this PR is adding this new setting to our defaults, in `get_default_provider_settings`